recipes/mgs-canopy/build.sh

```diff
@@ -0,0 +1,13 @@
+export LD_LIBRARY_PATH=$PREFIX
+cd src/
+
+echo "> Compiling sources"
+for F in $(find . -name "*.cpp") ; do
+	FF=${F%.*}
+	g++ -o $FF.o -fopenmp -c -Wall -Wextra -O3 -march=native -I./ -I${PREFIX}/include -L${PREFIX}/lib $FF.cpp
+done;
+
+echo "> Building target: cc.bin"
+g++ -o cc.bin -fopenmp -I${PREFIX}/include -L${PREFIX}/lib -lboost_program_options $(find . -name "*.o")
```

Comment:
This is the default name in the [Makefile](https://github.com/fplaza/mgs-canopy-algorithm/blob/master/src/Makefile#L33). I supposed people who are used to this software would look for `cc.bin`.
I can create a link to have `cc.bin` and `canopy` for example.